src/test/scala/mesosphere/mesos/TaskBuilderTest.scala

```diff
@@ -306,22 +306,21 @@ class TaskBuilderTest extends MarathonSpec with Matchers {
     assert(resource("cpus") == ScalarResource("cpus", 1)) // sanity, we DID match the offer, right?
 
     // check protobuf construction, should be a ContainerInfo w/ volumes
-    def vol(name: String): Option[MesosProtos.Volume] = {
+    def vol(path: String): Option[MesosProtos.Volume] = {
       if (taskInfo.hasContainer) {
-        taskInfo.getContainer.getVolumesList.asScala.find(_.getHostPath == name)
+        taskInfo.getContainer.getVolumesList.asScala.find(_.getHostPath == path)
       } else None
     }
 
     assert(taskInfo.getContainer.getVolumesList.size > 0, "check that container has volumes declared")
     assert(!taskInfo.getContainer.getDocker.hasVolumeDriver, "docker spec should not define a volume driver")
```

Comment:
Can't we remove this assertion (and the other similar ones), so that we can reenable the scalac `deprecated` check for tests?

`Docker.volumeDriver` is deprecated, so the scalac `deprecated` check ensures that it is not set by the "production" code.